[FIX] Payroll - Hong Kong Typo and Formatting (#18226)
odoo/documentation: content/applications/hr/payroll/payroll_localizations/hong_kong.rst

Reviewer: On line 250 of the doc, we refer to the field as "Sick Leave 80%".

> :guilabel:`Sick Leave 80%`: Paid sick leave compensated at 80% of the employee's average daily
  wage.

Which is the more UI-accurate term? sick or sickness leave? let's choose the one that's more aligned with the UI (:

Author: 'HK Sick Leaves' is the time off type, so I'll update all of them

     - The base module to manage the chart of accounts and localization for Hong Kong.
   * - :guilabel:`Hong Kong - Payroll`
     - `l10n_hk_hr_payroll`

>      - Enables payroll specific localization features for the Odoo Payroll app. This module also

Reviewer: ```suggestion
     - Enables payroll-specific localization features for the Odoo Payroll app. This module also
```

======================

Odoo supports multicompany management. First, the company must be configured. Navigate to
:menuselection:`Settings app --> Users & Companies --> Companies`. Create a new company or select an
existing one, and configure the following fields:

- :guilabel:`Company Name`: Enter the business name in this field.
- :guilabel:`Address`: Complete the full address, including the :guilabel:`City`, :guilabel:`State`,
  :guilabel:`Zip Code`, and :guilabel:`Country`.
- :guilabel:`Currency`: By default, `HKD` is selected. If not, select `HKD` from the drop-down menu.
- :guilabel:`Phone`: Enter the company phone number.
- :guilabel:`Email`: Enter the email used for general contact information.

.. note::
   The :guilabel:`Company Name` entered here automatically populates as the Employer's Name on

>    reports for IRD reporting. Only `HKD` is supported for compliant Hong Kong localization payroll

Reviewer: missing abbr for IRD!

salary rules:

- :guilabel:`Pre-transition Offset`: The amount of MPF contribution (or benefits from an :abbr:`ORSO
  (Occupational Retirement Schemes Ordinance)` scheme) that can be used to offset the pre-transition
  amount.
- :guilabel:`Post-transition Offset`: The amount of MPF contribution (or benefits from an
  :abbr:`ORSO (Occupational Retirement Schemes Ordinance)` scheme) that can be used to offset the
  post-transition amount.

.. _payroll/hk/work_entry_types:

Work entry type
---------------

Navigate to :menuselection:`Payroll app --> Configuration --> Work Entries --> Work Entry Types` to
check work entry types or configure a new work entry type.

In defining each work entry type for Hong Kong, employers can define:

- :guilabel:`Rate`: The rate that is used to calculate for each leave (e.g., 80% for sick leave).
- :guilabel:`ADW Calculation`: Enable this to calculate the employee's pay according to the Average
  Daily Wage (ADW) rules under Hong Kong employment regulations.

.. image:: hong_kong/work-entry-type.png
   :alt: Configuration screen for an 80 percent Sick Leave Work Entry Type.

> Odoo supports 11 work specific HK work entry types which are used together with the **Time Off**

Reviewer: not sure if it's the same thing but i think this is more concise?

```suggestion
Odoo supports 11 HK-specific work entry types which are used together with the **Time Off**
```

MPF schemes
-----------

> Odoo allows multiple MPF scheme configurations (currently 24 schemes), creation of new schemes,

Reviewer: abbr for MPF!

Reviewer: or if it's something really common for Hong Kong readers like PDF, please ignore. it might not need an abbreviation

Author: It's a "Mandatory Provident Fund" - so like an IRA over here. I think it's likley a well-known acronym (like we all know what an IRA is even though we never spell it out)- but I'll add it!

accuracy and catch errors. This includes checking that all time off is approved and any overtime is
appropriate.

Work entries sync based on the employee's contract configuration. Odoo pulls from the assigned
working schedule, attendance records, planning schedule, and approved time off. Any discrepancies or
conflicts must be resolved, then the work entries can be regenerated.

Once everything is correct, draft payslips can be created individually or in groups, referred to in
the **Payroll** app as pay runs.

.. note::
   If the work entry or salary inputs for an employee are amended, click the (gear) icon
   :menuselection:`--> Recompute Whole Sheet` to refresh the payslip's worked days and salary inputs
   tab.

Individual payslip
------------------

To create an individual payslip in the **Payroll** app:

Go to :menuselection:`Payroll app --> Payslips --> Payslips`. Click :guilabel:`New Off-Cycle` to
create a payslip for a single employee. Select the employee and period.

.. tip::
   Odoo automatically excludes out-of-contract days for pro-rata calculations (i.e., you do not need
   to select the period according to the employee joining date; Odoo automatically excludes

>    out-of-contract days and reduces from the basic salary).

Reviewer: ```suggestion
   out-of-contract days and reduces the basic salary).
```

   leave it blank.

Select employee(s). A pay run is created along with the drafted employee payslips. Check all
calculations and click the :guilabel:`Validate` button.

.. image:: hong_kong/batch-pay-run.png
   :alt: Validating a batch Pay Run.

.. important::
   An eMPF report is automatically generated after every pay run. Click the eMPF report or find it
   in :menuselection:`Reporting --> eMPF Contribution`.

.. _payroll/hk/accounting_check:

Accounting check
----------------

The accounting process when running payroll has two components: creating journal entries, and
registering payments. For each component, there are two possible scenarios: individual or batch.

Journal entry creation
~~~~~~~~~~~~~~~~~~~~~~

After payslips are confirmed and validated, journal entries are posted either individually or in a
batch. The journal entry is created first as a draft.

.. important::
   It must be decided if journal entries are done individually or in batches (:guilabel:`Batch
   Account Move Lines`) in the configuration before running payroll.

Individual journal entries
**************************

> Without enabling the batch account move line option, you need manual journal entries postings for

Reviewer: ```suggestion
Without enabling the batch account move line option, you need manual journal entry postings for
```

each payslip, regardless of using a pay run or individual payslip. Select a payslip and click the
:guilabel:`Journal Entry (Draft)` button.

.. important::
   The :guilabel:`Pay` button appears only for individual journal entries posting. Enabling the
   :guilabel:`Batch Account Move Lines` option causes this button to disappear.

Click the :guilabel:`Post` button. Each payslip is automatically posted as individual journal
entries in :menuselection:`Accounting --> Journal entries`.

.. image:: hong_kong/journal-entry.png
   :alt: Posting a drafted Journal Entry.

Four accounts from the Hong Kong CoA are included with the payroll localization:

- `221001 MPF Accrued (Employer)`: Accrue MPF benefits that are spent by employers.

> - `221002 MPF Withheld (Employee)`: Withheld MPF benefits that are paid by employees, which are paid

Reviewer: ```suggestion
- `221002 MPF Withheld (Employee)`: Withheld MPF benefits that are paid by employees, which will be paid
```

eMPF contribution reports can be generated in two ways:

> #. **Auto generated reports**: After every pay run, eMPF contributions reports can be found in

Reviewer: ```suggestion
#. **Auto-generated reports**: After every pay run, eMPF contributions reports can be found in
```

   :menuselection:`Payroll app --> eMPF Contributions`.

> #. **Manual generated reports**: eMPF contribution reports can be generated manually only after the

Reviewer: ```suggestion
#. **Manually generated reports**: eMPF contribution reports can be generated manually only after the
```

   :alt: The eMPF Contribution report screen.

.. important::

>    Each contribution line can be amendable (i.e., member status and termination payment type can be

Reviewer: I think it's more grammatically correct if we say

- "Each contribution line can be amended", or
- "Each contribution line is amendable"

   updated or deleted). However, relevant income and contributions cannot be edited. If the
   contribution line for an employee is amended, click the (gear) icon :menuselection:`--> Recompute
   contribution` to refresh the information.

Click the :guilabel:`Validate` button. If there are no errors, eMPF contribution reports are
generated in three different `.csv` files inside a zip file, which are ready to upload to the eMPF
portal: `new employees.csv`, `contributions.csv`, and `terminated employees.csv`.

.. important::

>    Warnings mainly raise to align with eMPF requirements for employee information. Employees must be

Reviewer: ```suggestion
   Warnings are mainly raised to align with eMPF requirements for employee information. Employees must be
```

Payslip tabs
============

Different tabs in each payslip define worked days used to calculate payroll, additional salary
inputs to add to the payslip, how salary computation is handled, and other info that impacts the
payslip.

- :guilabel:`Worked Days`: Details all the individual attendance records for the time period,
  including both worked time and any time off taken. It is auto-populated and cannot be edited.
- :guilabel:`Salary Inputs`: Details where additional salary inputs are listed, such as deductions,
  reimbursements, and expenses. Click :guilabel:`Add Inputs` to manually include them.
- :guilabel:`Salary Computation`: Where all the individual salary rules are listed and calculated,
  including everything from the employee's salary to all the deductions, taxes, and MPF
  contributions.
- :guilabel:`Other Info`: Shows the :guilabel:`Average Daily Wage` rate that is used to calculate

>   statutory HK holidays and entitlements. Also contains the :guilabel:`End-of-Year Pay` :

Reviewer: ```suggestion
  statutory HK holidays and entitlements. Also contains the :guilabel:`End-of-Year Pay`.
```

.. note::
   Manually adding an end-of-year payment to a specific payslip may require saving the record

>    manually, clicking the (gear) icon, and then selecting :guilabel:`Recompute Whole Sheet`.

Reviewer: might be missing the `:icon:` for gear here?

CAP 57: Long Service Payment
----------------------------

Calculate long service payment for Anthony Choi who joined Jan 1, 2021, with a monthly salary of 20K

> and leaves the company on Feb 28, 2026.

Reviewer: ```suggestion
and left the company on Feb 28, 2026.
```

the other scenarios were in past tense so maybe we stay consistent throughout?
